HISTORY.rst

```diff
@@ -7,6 +7,9 @@ Release History
 0.24.0
 +++++++++++++++
 
+**General updates**
+
+* Updated the minimum core CLI version to 2.46.0
```

Comment:
We need to update the metadata json file https://github.com/Azure/azure-iot-cli-extension/blob/dev/azext_iot/azext_metadata.json#L2

Reply:
done